runtime-testsuite/resources/org/antlr/v4/test/runtime/new_descriptors/CompositeParsers/BringInLiteralsFromDelegate.txt

```diff
@@ -0,0 +1,23 @@
+[type]
+CompositeParser
+
+[grammar]
+grammar M;
+import S;
+s : a ;
+WS : (' '|'\n') -> skip ;
+
+[slaveGrammar]
+parser grammar S;
+a : '=' 'a' {<write("\"S.a\"")>};
+
+[start]
+s
+
+[input]
+=a
+
+[output]
+"""S.a
+"""
```

Comment:
Is it possible to remove useless `"""` for almost all `output`? It looks untidy. It should be whitespace tolerant because it contains an output parse tree. The same for `errors`.

Reply:
For output newlines really matter because we might be generating that rather than simply calling println. For errors, it's true we can remove except then we'd have different rules for different sections of the file.